lib/ansible/modules/cloud/azure/azure_rm_subnet.py

```diff
@@ -68,6 +68,19 @@
             - The str can be the name or resource id of the route table.
             - The dict can contains C(name) and C(resource_group) of the route_table.
         version_added: "2.7"
+    service_endpoints:
```

Comment:
Should we add this as return value in facts module?

Reply:
yes, definitely

Reply:
in fact, it seems like we don't have azure_rm_subnet_facts module....
or i am blind....

Reply:
aha, it is in the virtual network module but in raw format

Reply:
hehe, which we should make obsolete :-)
but anyway, if it's raw, that means this info will be already there, just not documented.
do you think we should have subnet facts?
probably...
I don't think anyone wants to dig up subnet info from virtual network raw format